SDKTest/TrackerEventsTests.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  TrackerEventsTests.swift
+//  SDKTest
+//
+//  Created by Kevin Sites on 9/1/23.
+//
+
+@testable import NeuroID
+import XCTest
+
+class TouchEventTests: XCTestCase {
```

Comment:
Should the class be named TrackerEventsTests instead of TouchEventTests?

Reply:
Done, originally it was going to be a file with all the tracker event tests not just the touch ones, but the others didn't have any worthwhile tests to be written